src/opera_disp_tms/utils.py

```diff
@@ -16,6 +16,34 @@
 DATE_FORMAT = '%Y%m%dT%H%M%SZ'
 
 
+def list_files_in_s3(bucket: str, bucket_prefix: str):
+    resp = S3_CLIENT.list_objects_v2(Bucket=bucket, Prefix=bucket_prefix)
+
+    return resp['Contents']
+
+
+def download_file_from_s3(bucket: str, download_key, dest_dir: Path) -> Path:
+    """Download a file from an S3 bucket
+
+    Args:
+        uri: URI of the file to download
+        dest_dir: the directory to place the downloaded file in
+    """
```

Comment:
Docstr doesn't match function signature, but I don't think we need a docstr since it's a non-public function.
```suggestion
```